sdk/src/effect-serializer.js

```diff
@@ -19,9 +19,7 @@ const util = require("util");
 
 module.exports = class EffectSerializer {
 
-  constructor(allComponents) {
-    this.allComponents = allComponents;
-  }
+  constructor() {}
```

Comment:
Is it still needed if it has no parameters and doesn't do anything? Also, since it doesn't have any state anymore it could be a singleton rather than something you create an instance of when you want to use it?

Reply:
agree done 👍